app/api/tests/test_loginview.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+import json
+from app.api.tests.test_base import BaseTest
+
+
+class TestUser(BaseTest):
+	"""docstring for TestUser"""
+	def test_user_login(self):
+		respo= self.client.post('/api/v1/users/login',data = json.dumps(self.login_dummy_data),
```

Comment:
Use full variable names. For this case rename all appearances of `respo` to `response`